src/ViewRendererBundle/objects/helper/TPkgViewRendererSnippetDirectory.class.php

```diff
@@ -28,33 +28,22 @@ class TPkgViewRendererSnippetDirectory implements TPkgViewRendererSnippetDirecto
     const PATH_MODULES = 'webModules';
     const PATH_OBJECTVIEWS = 'objectviews';
     const PATH_LAYOUTS = 'layoutTemplates';
-    /**
-     * @var PortalDomainServiceInterface
-     */
-    private $portalDomainService;
-    /**
-     * @var RequestInfoServiceInterface
-     */
-    private $requestInfoService;
-    /**
-     * @var KernelInterface
-     */
-    private $kernel;
-    /**
-     * @var CmsConfigDataAccessInterface
-     */
-    private $cmsConfigDataAccess;
+
+    private RequestInfoServiceInterface $requestInfoService;
+    private KernelInterface $kernel;
+    private ThemeServiceInterface $themeService;
+    private PortalDomainServiceInterface $portalDomainService;
```

Comment:
Typed properties were introduced in php 7.4. Chameleon 7.0 has a minimum requirement of php 7.1. If you intend to use this fix in 7.0 then you need to undo this change.

Reply:
ok